extensions/eda/plugins/event_filter/json_filter.py

```diff
@@ -1,23 +1,39 @@
-"""json_filter.py:   An event filter that filters keys out of events.
-
-Includes override excludes.
-
-This is useful to exclude information from events that is unneeded by the rule
-engine.
-
-Arguments:
----------
-    * exclude_keys = a list of strings or patterns to remove
-    * include_keys = a list of strings or patterns to keep even if it matches
-    exclude_keys patterns.
-
-"""
-
 from __future__ import annotations
 
 import fnmatch
 from typing import Any, Optional
 
+DOCUMENTATION = r"""
+---
+author:
+  - Doston Toirov (@dtoirov)
+short_description: Filter keys out of events.
+description:
+  - An event filter that filters keys out of events.
+    Includes override excludes.
+    This is useful to exclude information from events that is unneeded by the rule engine.
+options:
+  exclude_keys:
+    description:
+      - A list of strings or patterns to remove.
+    type: list
+    elements: str
+    default: null
+  include_keys:
+    description:
+      - A list of strings or patterns to keep even if it matches exclude_keys patterns.
```

Comment:
@Dostonbek1 for botth the exclude_keys and include_keys we have to mention that it is a path as shown here https://github.com/mkanoor/eda-project/blob/e4a8170c8fb2565e4633673694c4bb83b4c1a8e0/rulebooks/webhook_with_filter.yml#L20 so if the data is under key1.key2.key3 it would have to be listed as 
- key1
- key2
- key3
Since they are nested path, we can't just exclude key3 it needs to be the full path.

Another example https://github.com/ansible/ansible-rulebook/blob/b4f8b6c69be2d84f3e6fc6bd61cd0dab41b2ac43/tests/examples/60_json_filter.yml#L5

Reply:
pushed the suggested changes.